### PR TITLE
Edit dct

### PR DESCRIPTION
**Pull request by Eebru-gzy**
This PR changes the DCT environment variable to 0 when it is pushing the SCIM image

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -210,7 +210,7 @@ jobs:
           SERVICE_NAME: ${{ steps.setup.outputs.service_name }}
         run: |
           if [ "$SERVICE_NAME" == "scim" ]; then
-            $DOCKER_CONTENT_TRUST==0 
+            DOCKER_CONTENT_TRUST==0
```
Comment (joseph-flinn) [inline .github/workflows/release.yml:213]: Did the `export` here not work?

Comment (joseph-flinn) [reply .github/workflows/release.yml:213]:
```suggestion
            export DOCKER_CONTENT_TRUST=0
```

Comment (michalchecinski) [reply .github/workflows/release.yml:213]: I believe before it was `export $DOCKER_CONTENT_TRUST=0`. I messed the `$` during review

[the remark this answers is not in the document]
Comment (joseph-flinn) [reply .github/workflows/release.yml]: I think this either needs to be `export DOCKER_CONTENT_TRUST=0` or a separate step in the workflow with a conditional

Comment (Eebru-gzy) [reply .github/workflows/release.yml]: Yes, it is updated now

Comment (joseph-flinn) [reply .github/workflows/release.yml]: @Eebru-gzy It looks like it is still missing the `export`

**Review by michalchecinski — APPROVED** [review]

**Review by joseph-flinn — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (Eebru-gzy) [reply .github/workflows/release.yml]: Added now

**Pull request merged**